lib/ordered_map.h

```diff
@@ -36,7 +36,7 @@ class ordered_map {
     typedef const value_type            &const_reference;
 
  private:
-    typedef std::list<value_type, ALLOC>                list_type;
+    typedef std::list<value_type>                       list_type;
```

Comment:
This is unnecessary -- the allocator was correct

Reply:
Where is this file？And how can I find this file?